src/voprf.rs

```diff
@@ -600,7 +600,23 @@ impl<G: Group, H: BlockInput + Digest> BlindedElement<G, H> {
     #[cfg(test)]
     /// Only used for testing zeroize
     pub fn as_ptrs(&self) -> Vec<Vec<u8>> {
-        vec![self.value.to_arr().to_vec()]
+        alloc::vec![self.value.to_arr().to_vec()]
+    }
+
+    #[cfg(feature = "internal")]
+    /// Unsafely set the value. This should be used with caution, since
+    /// it does not perform any checks on the validity of the value itself!
```

Comment:
```suggestion
    /// Create a `BlindedElement`.
    ///
    /// # Dangerous
    /// This should be used with caution, since it does not perform any
    /// checks on the validity of the value itself!
```
Typically a markdown header is used for this kind of thing.